backend/plugin/db/util/driverutil.go

```diff
@@ -118,6 +118,59 @@ func ApplyMultiStatements(sc io.Reader, f func(string) error) error {
 	return nil
 }
 
+// QueryV2 is a copy of Query, but do not mask the data(use none masker).
```

Comment:
Do we need a copy for QueryV2? Supposely passing nil column info will disable the masking?

Reply:
Let's keep it simple to lighten the mental load.